README.md

```diff
@@ -1,152 +1,85 @@
-#Twitter Cleo App
+# Twitter Cleo App
 
-## Readme format
+### Readme format
 
 The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)
 
+### Fork strategy: [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+ - Main branch: main
+ - Feature branch: feature/XXX-XXX-XXX-name-ticket
 
-
-* Roles:
-- * Podruct Owner
-- * Technical Lead
-- * Facilitator
-
-* Fork strategy
-- * [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
-- * [Git flow](https://www.atlassian.com/es/git/tutorials/comparing-workflows/gitflow-workflow#:~:text=Gitflow%20es%20un%20modelo%20alternativo,vez%20y%20quien%20lo%20populariz%C3%B3)
-
```

Comment:
Maybe would be nice to put in the readme which one you will use.

Reply:
### Readme format

The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)

Reply:
### Readme format

The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)

Reply:
What @sebastianMullerEndava was referring is to only put in the readme the Git/Fork strategy that you are going to use for this project and not both as you did here

Reply:
The following were added:
Fork strategy: [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
Main branch: main
Feature branch: feature/XXX-XXX-XXX-name-ticket

Reply:
please change "fork strategy" to "branch strategy"

Reply:
It's done